WordPress.xcworkspace/contents.xcworkspacedata

```diff
@@ -1,6 +1,9 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <Workspace
    version = "1.0">
+   <FileRef
```

Comment:
Let's add `Modules` to a project and not a workspace. We we going to remove the workspace.

Reply:
Done.

Reply:
> We we going to remove the workspace.

I started looking at it https://github.com/wordpress-mobile/WordPress-iOS/pull/24534